proxy/http/HttpTransact.cc

```diff
@@ -63,7 +63,12 @@ static char range_type[] = "multipart/byteranges; boundary=RANGE_SEPARATOR";
 
 extern HttpBodyFactory *body_factory;
 
-static const char local_host_ip_str[] = "127.0.0.1";
+inline static bool
+is_localhost(const char *name, int len)
```

Comment:
Since we did this, maybe this should actually be part of lib/ts? e.g. ts_is_localhost() or some such ? There's at least one other place where a similar comparison is done:
```c
proxy/http/HttpSM.cc:    if ((strncmp(t_state.dns_info.lookup_name, "127.0.0.1", 9) == 0 || strncmp(t_state.dns_info.lookup_name, "::1", 3) == 0) &&